src/benchmarks/micro/libraries/System.Numerics.Vectors/Perf_Matrix3x2.cs

```diff
@@ -0,0 +1,110 @@
+        public Matrix3x2 MultiplyByScalarOperatorBenchmark() => Matrix3x2.Identity * 0.5f;
+
+        [Benchmark]
+        public Matrix3x2 SubtractOperatorBenchmark() => Matrix3x2.Identity - Matrix3x2.Identity;
+
+        [Benchmark]
+        public Matrix3x2 NegationOperatorBenchmark() => -Matrix3x2.Identity;
+
+        [Benchmark]
+        public Matrix3x2 AddBenchmark() => Matrix3x2.Add(Matrix3x2.Identity, Matrix3x2.Identity);
+
+        [Benchmark]
+        public Matrix3x2 CreateRotationBenchmark() => Matrix3x2.CreateRotation(PI / 2.0f);
+
+        [Benchmark]
+        public Matrix3x2 CreateRotationWithCenterBenchmark() => Matrix3x2.CreateRotation(PI / 2.0f, Vector2.Zero);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromScalarXYBenchmark() => Matrix3x2.CreateScale(1.0f, 2.0f);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromScalarWithCenterBenchmark() => Matrix3x2.CreateScale(1.0f, Vector2.Zero);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromScalarXYWithCenterBenchmark() => Matrix3x2.CreateScale(1.0f, 2.0f, Vector2.Zero);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromScalarBenchmark() => Matrix3x2.CreateScale(1.0f);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromVectorBenchmark() => Matrix3x2.CreateScale(Vector2.UnitX);
+
+        [Benchmark]
+        public Matrix3x2 CreateScaleFromVectorWithCenterBenchmark() => Matrix3x2.CreateScale(Vector2.UnitX, Vector2.Zero);
+
+        [Benchmark]
+        public Matrix3x2 CreateSkewFromScalarXYBenchmark() => Matrix3x2.CreateSkew(1.0f, 2.0f);
+
+        [Benchmark]
+        public Matrix3x2 CreateSkewFromScalarXYWithCenterBenchmark() => Matrix3x2.CreateSkew(1.0f, 2.0f, Vector2.Zero);
+
+        [Benchmark]
+        public Matrix3x2 CreateTranslationFromVectorBenchmark() => Matrix3x2.CreateTranslation(Vector2.UnitX);
+
+        [Benchmark]
+        public Matrix3x2 CreateTranslationFromScalarXY() => Matrix3x2.CreateTranslation(1.0f, 2.0f);
+
+        [Benchmark]
+        public bool EqualsBenchmark() => Matrix3x2.Identity.Equals(Matrix3x2.Identity);
+
+        [Benchmark]
+        public float GetDeterminantBenchmark() => Matrix3x2.Identity.GetDeterminant();
+
+        [Benchmark]
+        public bool InvertBenchmark() => Matrix3x2.Invert(Matrix3x2.Identity, out Matrix3x2 result);
+
+        [Benchmark]
+        public Matrix3x2 LerpBenchmark() => Matrix3x2.Lerp(default, Matrix3x2.Identity, 0.5f);
+
+        [Benchmark]
+        public Matrix3x2 MultiplyByMatrixBenchmark() => Matrix3x2.Multiply(Matrix3x2.Identity, Matrix3x2.Identity);
+
+        [Benchmark]
+        public Matrix3x2 MultiplyByScalarBenchmark() => Matrix3x2.Multiply(Matrix3x2.Identity, 0.5f);
+
+        [Benchmark]
+        public Matrix3x2 NegateBenchmark() => Matrix3x2.Negate(Matrix3x2.Identity);
+
+        [Benchmark]
+        public Matrix3x2 SubtractBenchmark() => Matrix3x2.Subtract(Matrix3x2.Identity, Matrix3x2.Identity);
+    }
```

Comment:
all these benchmarks access `Matrix3x2.Identity` (static property that returns static field) and typically perform very short operations like add/multiply few numbers. The reported time will likely be dominated by static field access (it's not as cheap as it used to be in the past).

Example:

```cs
public class StaticOrNot
{
    private Matrix3x2 _field = Matrix3x2.Identity;

    [Benchmark]
    public Matrix3x2 SubtractBenchmark_Static() => Matrix3x2.Subtract(Matrix3x2.Identity, Matrix3x2.Identity);

    [Benchmark]
    public Matrix3x2 SubtractBenchmark_Instance() => Matrix3x2.Subtract(_field, _field);
}
```

|                     Method |      Mean |
|--------------------------- |----------:|
|   SubtractBenchmark_Static | 10.062 ns |
| SubtractBenchmark_Instance |  7.810 ns |

@tannergooding what do you think about adding an instance field to this type and using it instead of the static property?

Codegen below:

<details>

## .NET Core 5.0.0 (CoreCLR 5.0.20.42308, CoreFX 5.0.20.42308), X64 RyuJIT
```assembly
; MicroBenchmarks.Matrix.SubtractBenchmark_Static()
       7FFA2EA73B70 push      rsi
       7FFA2EA73B71 sub       rsp,80
       7FFA2EA73B78 vzeroupper
       7FFA2EA73B7B mov       rsi,rdx
       7FFA2EA73B7E lea       rcx,[rsp+68]
       7FFA2EA73B83 call      System.Numerics.Matrix3x2.get_Identity()
       7FFA2EA73B88 lea       rcx,[rsp+50]
       7FFA2EA73B8D call      System.Numerics.Matrix3x2.get_Identity()
       7FFA2EA73B92 mov       rcx,rsi
       7FFA2EA73B95 vmovdqu   xmm0,xmmword ptr [rsp+68]
       7FFA2EA73B9B vmovdqu   xmmword ptr [rsp+38],xmm0
       7FFA2EA73BA1 mov       rdx,[rsp+78]
       7FFA2EA73BA6 mov       [rsp+48],rdx
       7FFA2EA73BAB vmovdqu   xmm0,xmmword ptr [rsp+50]
       7FFA2EA73BB1 vmovdqu   xmmword ptr [rsp+20],xmm0
       7FFA2EA73BB7 mov       rdx,[rsp+60]
       7FFA2EA73BBC mov       [rsp+30],rdx
       7FFA2EA73BC1 lea       rdx,[rsp+38]
       7FFA2EA73BC6 lea       r8,[rsp+20]
       7FFA2EA73BCB call      System.Numerics.Matrix3x2.Subtract(System.Numerics.Matrix3x2, System.Numerics.Matrix3x2)
       7FFA2EA73BD0 mov       rax,rsi
       7FFA2EA73BD3 add       rsp,80
       7FFA2EA73BDA pop       rsi
       7FFA2EA73BDB ret
; Total bytes of code 108
```
```assembly
; System.Numerics.Matrix3x2.get_Identity()
       7FFA2EA73A80 vzeroupper
       7FFA2EA73A83 mov       rax,120AB8B1440
       7FFA2EA73A8D mov       rax,[rax]
       7FFA2EA73A90 vmovdqu   xmm0,xmmword ptr [rax+8]
       7FFA2EA73A95 vmovdqu   xmmword ptr [rcx],xmm0
       7FFA2EA73A99 mov       rdx,[rax+18]
       7FFA2EA73A9D mov       [rcx+10],rdx
       7FFA2EA73AA1 mov       rax,rcx
       7FFA2EA73AA4 ret
; Total bytes of code 37
```
```assembly
; System.Numerics.Matrix3x2.Subtract(System.Numerics.Matrix3x2, System.Numerics.Matrix3x2)
       7FFA2EA73AC0 sub       rsp,18
       7FFA2EA73AC4 vzeroupper
       7FFA2EA73AC7 vmovss    xmm0,dword ptr [rdx]
       7FFA2EA73ACB vsubss    xmm0,xmm0,dword ptr [r8]
       7FFA2EA73AD0 vmovss    dword ptr [rsp],xmm0
       7FFA2EA73AD5 vmovss    xmm0,dword ptr [rdx+4]
       7FFA2EA73ADA vsubss    xmm0,xmm0,dword ptr [r8+4]
       7FFA2EA73AE0 vmovss    dword ptr [rsp+4],xmm0
       7FFA2EA73AE6 vmovss    xmm0,dword ptr [rdx+8]
       7FFA2EA73AEB vsubss    xmm0,xmm0,dword ptr [r8+8]
       7FFA2EA73AF1 vmovss    dword ptr [rsp+8],xmm0
       7FFA2EA73AF7 vmovss    xmm0,dword ptr [rdx+0C]
       7FFA2EA73AFC vsubss    xmm0,xmm0,dword ptr [r8+0C]
       7FFA2EA73B02 vmovss    dword ptr [rsp+0C],xmm0
       7FFA2EA73B08 vmovss    xmm0,dword ptr [rdx+10]
       7FFA2EA73B0D vsubss    xmm0,xmm0,dword ptr [r8+10]
       7FFA2EA73B13 vmovss    dword ptr [rsp+10],xmm0
       7FFA2EA73B19 vmovss    xmm0,dword ptr [rdx+14]
       7FFA2EA73B1E vsubss    xmm0,xmm0,dword ptr [r8+14]
       7FFA2EA73B24 vmovss    dword ptr [rsp+14],xmm0
       7FFA2EA73B2A vmovdqu   xmm0,xmmword ptr [rsp]
       7FFA2EA73B2F vmovdqu   xmmword ptr [rcx],xmm0
       7FFA2EA73B33 mov       rax,[rsp+10]
       7FFA2EA73B38 mov       [rcx+10],rax
       7FFA2EA73B3C mov       rax,rcx
       7FFA2EA73B3F add       rsp,18
       7FFA2EA73B43 ret
; Total bytes of code 132
```

## .NET Core 5.0.0 (CoreCLR 5.0.20.42308, CoreFX 5.0.20.42308), X64 RyuJIT
```assembly
; MicroBenchmarks.Matrix.SubtractBenchmark_Instance()
       7FFA2EA83A70 push      rsi
       7FFA2EA83A71 sub       rsp,50
       7FFA2EA83A75 vzeroupper
       7FFA2EA83A78 mov       rsi,rdx
       7FFA2EA83A7B mov       rdx,rsi
       7FFA2EA83A7E add       rcx,8
       7FFA2EA83A82 vmovdqu   xmm0,xmmword ptr [rcx]
       7FFA2EA83A86 vmovdqu   xmmword ptr [rsp+38],xmm0
       7FFA2EA83A8C mov       r8,[rcx+10]
       7FFA2EA83A90 mov       [rsp+48],r8
       7FFA2EA83A95 vmovdqu   xmm0,xmmword ptr [rcx]
       7FFA2EA83A99 vmovdqu   xmmword ptr [rsp+20],xmm0
       7FFA2EA83A9F mov       r8,[rcx+10]
       7FFA2EA83AA3 mov       [rsp+30],r8
       7FFA2EA83AA8 mov       rcx,rdx
       7FFA2EA83AAB lea       rdx,[rsp+38]
       7FFA2EA83AB0 lea       r8,[rsp+20]
       7FFA2EA83AB5 call      System.Numerics.Matrix3x2.Subtract(System.Numerics.Matrix3x2, System.Numerics.Matrix3x2)
       7FFA2EA83ABA mov       rax,rsi
       7FFA2EA83ABD add       rsp,50
       7FFA2EA83AC1 pop       rsi
       7FFA2EA83AC2 ret
; Total bytes of code 83
```
```assembly
; System.Numerics.Matrix3x2.Subtract(System.Numerics.Matrix3x2, System.Numerics.Matrix3x2)
       7FFA2EA83AE0 sub       rsp,18
       7FFA2EA83AE4 vzeroupper
       7FFA2EA83AE7 vmovss    xmm0,dword ptr [rdx]
       7FFA2EA83AEB vsubss    xmm0,xmm0,dword ptr [r8]
       7FFA2EA83AF0 vmovss    dword ptr [rsp],xmm0
       7FFA2EA83AF5 vmovss    xmm0,dword ptr [rdx+4]
       7FFA2EA83AFA vsubss    xmm0,xmm0,dword ptr [r8+4]
       7FFA2EA83B00 vmovss    dword ptr [rsp+4],xmm0
       7FFA2EA83B06 vmovss    xmm0,dword ptr [rdx+8]
       7FFA2EA83B0B vsubss    xmm0,xmm0,dword ptr [r8+8]
       7FFA2EA83B11 vmovss    dword ptr [rsp+8],xmm0
       7FFA2EA83B17 vmovss    xmm0,dword ptr [rdx+0C]
       7FFA2EA83B1C vsubss    xmm0,xmm0,dword ptr [r8+0C]
       7FFA2EA83B22 vmovss    dword ptr [rsp+0C],xmm0
       7FFA2EA83B28 vmovss    xmm0,dword ptr [rdx+10]
       7FFA2EA83B2D vsubss    xmm0,xmm0,dword ptr [r8+10]
       7FFA2EA83B33 vmovss    dword ptr [rsp+10],xmm0
       7FFA2EA83B39 vmovss    xmm0,dword ptr [rdx+14]
       7FFA2EA83B3E vsubss    xmm0,xmm0,dword ptr [r8+14]
       7FFA2EA83B44 vmovss    dword ptr [rsp+14],xmm0
       7FFA2EA83B4A vmovdqu   xmm0,xmmword ptr [rsp]
       7FFA2EA83B4F vmovdqu   xmmword ptr [rcx],xmm0
       7FFA2EA83B53 mov       rax,[rsp+10]
       7FFA2EA83B58 mov       [rcx+10],rax
       7FFA2EA83B5C mov       rax,rcx
       7FFA2EA83B5F add       rsp,18
       7FFA2EA83B63 ret
; Total bytes of code 132
```

</details>

Reply:
As per https://github.com/dotnet/performance/pull/1448#discussion_r479473329, `Identity` is a special property which will be getting codegen improvements in .NET 6. It is a core property that is meant to be fast and which may have specialized JIT handling such that it isn't actually a property returning a static field.